hw/mcu/nxp/src/ext/nxp-kinetis-sdk/drivers/fsl_ftfx_controller.c

```diff
@@ -516,7 +517,9 @@ status_t FTFx_CMD_Program(ftfx_config_t *config, uint32_t start, const uint8_t *
         }
 
         /* calling flash command sequence function to execute the command */
+        OS_ENTER_CRITICAL(sr);
         returnCode = ftfx_command_sequence(config);
+        OS_EXIT_CRITICAL(sr);
 

```

Comment:
I would suggest to move this file somewhere else so we don't have to maintain changes to this when we update the SDK.

Reply:
Done.